### PR TITLE
Feature docker compose code quality game

### PR DESCRIPTION
**Pull request by ricardogarfe**
Create docker compose file for code quality game and docker commands to run app

### CODE REVIEW

**Comment by mechero** [comment]
Thanks for the contribution! I'll have a look as soon as I have time.

**Comment by ricardogarfe** [comment]
thanks to you! great job ;)

File: code-quality-game-docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  code-quality-game:
+    image: maven:3.5.2-jdk-8
+    command: mvn spring-boot:run
+    expose:
+      - "8080"
+    ports:
+      - "8080:8080"
+    volumes:
+      - ~/.m2-docker:/root/.m2
+      - $PWD/sonar-connector:/usr/src/
```
Comment (mechero) [inline code-quality-game-docker-compose.yml]: `$PWD` environment variable is not available on Windows, but I'll change this.

File: code-quality-game-docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  code-quality-game:
+    image: maven:3.5.2-jdk-8
+    command: mvn spring-boot:run
+    expose:
+      - "8080"
+    ports:
+      - "8080:8080"
+    volumes:
+      - ~/.m2-docker:/root/.m2
```
Comment (mechero) [inline code-quality-game-docker-compose.yml]: I'm not sure about this one, why is it needed?

Comment (ricardogarfe) [reply code-quality-game-docker-compose.yml]: hi, sorry for being late, I used to have m2 path inside docker in another path different from the user to not being corrupted (user root or docker writes inside m2 default path and could break your local maven repository)

**Pull request merged**